codes/RequestTemplate.ttl

```diff
@@ -19,7 +19,7 @@ idsc:RequestTemplate a ids:PolicyTemplate ;
 		Optional: This only holds if <Attribute URI> is [equal|lower than|greater than] the [<Value URI>|<Value Literal>].
 		Optional: This only holds if also <Attribute URI> is [equal|lower than|greater than] the [<Value URI>|<Value Literal>]."""@en ;
     skos:note """{ 
-		"@context": "http://www.w3id.org/ids/contract.jsonld", 
+		"@context": "https://w3id.org/idsa/context.json",
```

Comment:
"The requested URL was not found on this server." - Will this be resolved by this pull request? If so, then you can ignore this comment